pkg/webhooks/imagecontentpolicies/imagecontentpolicies.go

```diff
@@ -208,3 +248,17 @@ func authorizeImageContentSourcePolicy(icsp operatorv1alpha1.ImageContentSourceP
 
 	return true
 }
+
+// isAllowedUserGroup checks if the user or group is allowed to perform the action
+func isAllowedUserGroup(request admission.Request) bool {
```

Comment:
Should we allow the backplane user to do this or a particular service account?

Reply:
Updated to only allow for SA